src/olympia/reviewers/serializers.py

```diff
@@ -348,25 +349,29 @@ def get_entries(self, obj):
             # Now we can set the git-status.
             entries[path]['status'] = patch['mode']
 
-            parent_path = os.path.dirname(path)
-            if (
-                path_deleted is True and
-                parent_path != '' and
-                parent_path not in entries
-            ):
-                # The parent directory of this deleted file does not
-                # exist. This could happen if no other files were
-                # modified within the directory.
-                entries[parent_path] = {
-                    'depth': path_depth - 1,
-                    'filename': os.path.basename(parent_path),
-                    'sha256': None,
-                    'mime_category': 'directory',
-                    'mimetype': 'application/octet-stream',
-                    'path': parent_path,
-                    'size': None,
-                    'modified': None,
-                }
+            parents = [x for x in pathlib.Path(path).parents if str(x) != '.']
```

Comment:
shouldn't `x` be renamed to something else more explicit?

Reply:
Indeed, I think e25bce37f6 will actually fix this in a better way. Let's see if I broke something accidentally though.